estela-web/src/components/ProjectCronJobListPage/index.tsx

```diff
@@ -324,9 +326,11 @@ export class ProjectCronJobListPage extends Component<RouteComponentProps<RouteP
         const requestParams: ApiProjectsSpidersListRequest = { pid: this.projectId, page, pageSize: this.PAGE_SIZE };
         this.apiService.apiProjectsSpidersList(requestParams).then(
             (results) => {
-                const spiders: Spider[] = results.results;
-                this.setState({ spiders: [...spiders] });
-                this.setState({ spiderId: String(results.results[0].sid) });
+                this.setState({
+                    spiders: [...results.results],
+                    spiderId: String(results.results[0].sid),
```

Comment:
Consider the case when the result is empty.

Reply:
Done